packages/carbon-graphs/tests/unit/controls/PairedResult/PairedResultLoad-spec.js

```diff
@@ -367,6 +367,88 @@ describe('Paired Result - Load', () => {
+      disableCalibrationInput.axis.x.lowerLimit = 500;
+      const disableCalibrationGraph = new Graph(disableCalibrationInput);
+      input = getInput(valuesDefault, false, false);
+      disableCalibrationGraph.loadContent(new PairedResult(input));
+
+      expect(disableCalibrationGraph.config.axis.x.domain.upperLimit)
+        .toEqual(disableCalibrationInput.axis.x.upperLimit);
+      expect(disableCalibrationGraph.config.axis.x.domain.lowerLimit)
+        .toEqual(disableCalibrationInput.axis.x.lowerLimit);
+    });
+    it('does not update x axis range if allow calibration is undefined', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
+      disableCalibrationInput.axis.x.lowerLimit = 500;
+      const disableCalibrationGraph = new Graph(disableCalibrationInput);
+      input = getInput(valuesDefault, false, false);
+      disableCalibrationGraph.loadContent(new PairedResult(input));
+
+      expect(disableCalibrationGraph.config.axis.x.domain.upperLimit)
+        .toEqual(disableCalibrationInput.axis.x.upperLimit);
+      expect(disableCalibrationGraph.config.axis.x.domain.lowerLimit)
+        .toEqual(disableCalibrationInput.axis.x.lowerLimit);
+    });
+    it('does not update x axis range if allow calibration is enabled and datapoints are within limits', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
+      disableCalibrationInput.axis.x.allowCalibration = true;
+      disableCalibrationInput.axis.x.lowerLimit = 0;
+      disableCalibrationInput.axis.x.upperLimit = 500;
+      const disableCalibrationGraph = new Graph(disableCalibrationInput);
+      input = getInput(valuesDefault, false, false);
+      disableCalibrationGraph.loadContent(new PairedResult(input));
+
+      expect(disableCalibrationGraph.config.axis.x.domain.upperLimit).toEqual(500);
+      expect(disableCalibrationGraph.config.axis.x.domain.lowerLimit).toEqual(0);
+    });
+    it('updates x axis range if allow calibration is enabled and datapoints are within limits', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
+      disableCalibrationInput.axis.x.allowCalibration = true;
+      disableCalibrationInput.axis.x.lowerLimit = 30;
+      disableCalibrationInput.axis.x.upperLimit = 35;
+      const disableCalibrationGraph = new Graph(disableCalibrationInput);
+      input = getInput(
+        [
+          {
+            high: {
+              x: 25,
+              y: 350,
+            },
+            mid: {
+              x: 25,
+              y: 146,
+            },
+            low: {
+              x: 25,
+              y: 75,
+            },
+          },
+          {
+            high: {
+              x: 45,
+              y: 110,
+            },
+            mid: {
+              x: 45,
+              y: 70,
+            },
+            low: {
+              x: 45,
+              y: 30,
+            },
+          },
+        ],
+        false,
+        false,
+      );
```

Comment:
There should be an existing dataset you can use (`valuesDefault`). It's in `packages/carbon-graphs/tests/unit/controls/PairedResult/helpers.js`

Reply:
[4934301](https://github.com/cerner/terra-graphs/pull/289/commits/4934301932e7aefa479cebbc85fb9a9e0179a1db)